managers/blockchain-operations-manager.js

```diff
@@ -6,11 +6,23 @@ class BlockchainOperationsManager {
         this.inputService = services.inputService;
     }
 
+    /**
+     * Retrieve the current gas price.
+     * @async
+     * @param {Object} [options={}]  - Optional parameters for blockchain service.
+     * @returns {Promise<string>} - A promise that resolves to the current gas price.
+     */
     async getGasPrice(options = {}){
         const blockchain = this.inputService.getBlockchain(options);
         return this.blockchainService.getGasPrice(blockchain);
     }
 
+    /**
+     * Retrieve the wallet balances.
+     * @async
+     * @param {Object} [options={}] - Optional parameters for blockchain service.
+     * @returns {Promise<Object>} - A promise that resolves to an object containing wallet balances.
```

Comment:
It is inconsistent to return Object at some places (like the function is not async) and Promise at others.. We should use one or another - I see in our code we also return Promises in query and info functions. I guess it would be amazing if we changed it to Promise<Object> and similar everywhere but since we're already so deep in with having returns Object (like the function is not async) we might as well go with that, we can discuss this

Reply:
We should separately discuss that